### PR TITLE
Adding the API index file

### PR DESCRIPTION
**Pull request by Sunitashelar**
Adding changes made to the API index file

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

File: apis/index.md [hunk 1]
```diff
@@ -27,109 +27,112 @@ Currently, Sunbird has the following microservice categories. Each microservice
...
 
 ## Practice, Sensing & Assessment Services
 
-The **Practice, Sensing & Assessment Services** category has the following microservice:
-
-- [**Assessment Service:**](apis/assessmentapi/){:target="_blank"} Provides services to create and manage questions and question sets that are useful to assess proficiency of learners.  
+The **Practice, Sensing & Assessment Services** category has a microservice that creates and manages questions and question sets that are useful to assess proficiency of learners. The APIs for this microservice are grouped under:
 
-<!--- [**Assessment Management Service:**](apis/itemsetapi/){:target="_blank"} (Under development) Provides services to create and manage questions and question sets that are useful to assess proficiency of learners.  -->
+- [Assessment Item APIs](apis/assessmentapi/){:target="_blank"}
```
Comment (mohitga) [inline apis/index.md:111]: Assessment API will be added in 3.7  release

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Sunitashelar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://712.qa.docs.sunbird.org/3.6.0/ .<br/> **Note:** Only _3.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://712.qa.docs.sunbird.org/3.6.0/apis// <br /> - http://712.qa.docs.sunbird.org/3.6.0/apis// <br />  <br/> **Modified sections are -** <br />

File: apis/index.md [hunk 2]
```diff
@@ -27,109 +27,112 @@ Currently, Sunbird has the following microservice categories. Each microservice
...
 
-- **Content Management:**  Provides services to manage content objects and their lifecycle for a defined taxonomy in a learning domain. This service is dependent on the taxonomy microservice as every content piece must be tagged with the appropriate taxonomy.  
-        [Content APIs](apis/content/){:target="_blank"}         
-        [Textbook APIs](apis/tocapi/){:target="_blank"}  
-<!--- add lock and plugin API folders and here --> 
-        
+- **Content Management:**  The content management microservice manages content objects and their lifecycle for a defined taxonomy, in a learning domain. This microservice is dependent on the taxonomy microservice as every content piece must have the appropriate taxonomy tags. The content management microservices have the following APIs:     
+        - [Content APIs](apis/content/){:target="_blank"}         
+        - [Textbook APIs](apis/tocapi/){:target="_blank"} 
+        - [Course Hierarchy APIs](apis/coursehierarchyapi//){:target="_blank"}
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply apis/index.md]: Done

File: apis/index.md [hunk 3]
```diff
@@ -27,109 +27,112 @@ Currently, Sunbird has the following microservice categories. Each microservice
...
+        - [Content Lock APIs](apis/lockService//){:target="_blank"} 
 
-- [**Usage & Tracking Service**](apis/courseprogressapi/){:target="_blank"}: Provides services for [enrolment](apis/courseenrolmentapi/){:target="_blank"}, usage and progress tracking of trackable learning objects like online courses, [course batches](apis/coursebatchmanapi/){:target="_blank"}, quizzes, [batch certificates](apis/coursebatchcertificateapi/){:target="_blank"}.
+- **Usage & Tracking Service**: The usage and tracking microservice is used for course enrolment, manage content usage, and to track progress of trackable learning objects like online courses, course batches, quizzes, and batch certificates. The APIs for this microservice are grouped into:
+        - [Course Progress APIs](apis/courseprogressapi/){:target="_blank"}
+        - [Course Enrolment APIs](apis/courseenrolmentapi/){:target="_blank"}
+        - [Course Batch Management APIs](apis/coursebatchmanapi/){:target="_blank"}
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply apis/index.md]: Done

File: apis/index.md [hunk 4]
```diff
@@ -27,109 +27,112 @@ Currently, Sunbird has the following microservice categories. Each microservice
...
+        - [Registry APIs](apis/opensaber/){:target="_blank"}
 
 ## Taxonomy & Tagging Services
 
-The **Taxonomy & Tagging Services** category has the following microservices:
+The **Taxonomy & Tagging Services** category has microservices for taxonomy and framework management, and content category management.  
 
-- [**Taxonomy Framework Management:**](apis/framework/){:target="_blank"} Provides the ability to create, organise and manage taxonomy frameworks in a learning domain using Sunbird platform APIs ([license APIs](apis/license/){:target="_blank"}). This microservice is domain independent, i.e. can be used to model the taxonomy of any learning domain, e.g: K-12, Leadership training, Professional training.  
+- **Taxonomy Framework Management:** The taxonomy and framework microservice provides the ability to create, organise and manage taxonomy frameworks in a learning domain using the license APIs from the Sunbird platform APIs. This microservice is domain independent, i.e. it can be used to model the taxonomy of any learning domain, e.g: K-12, Leadership training, Professional training. The APIs in this microservice are grouped as:
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply apis/index.md]: Done

**Pull request merged**